README.md

```diff
@@ -67,9 +67,21 @@ to revise it in response.
 * Build consensus and integrate feedback. RFCs that have broad support
 are much more likely to make progress than those that don't receive any
 comments.
-* Eventually, somebody on the [core team] will either accept the RFC by
-merging the pull request and assigning the RFC a number, at which point
-the RFC is 'active', or reject it by closing the pull request.
+* Eventually, the [core team] will decide whether the RFC is a candidate
+for inclusion in Ember.
+* RFCs that are candidates for inclusion in Ember will enter a "final comment
+period", which shall be no shorter than 7 days. This period will be signaled
+with a comment and tag on the RFC's pull request. Furthermore,
+[Ember's official Twitter account](https://twitter.com/emberjs) will post a
+tweet about the RFC to attract the community's attention.
+* At the close of the final comment period, the [core team] will decide whether
+to accept the RFC by merging the pull request and assigning the RFC a number,
```

Comment:
I wonder if we need to mention the concept of a number here at all? (I just copied this part from the old verbiage)